packages/core/admin/admin/src/content-manager/pages/ListView/index.js

```diff
@@ -486,6 +429,11 @@ function ListView({
   // Add 1 column for the checkbox and 1 for the actions
   const colCount = tableHeaders.length + 2;
 
+  // We have this function to refetch data when selected entries modal is closed
+  const refetchData = () => {
+    fetchData(`/content-manager/collection-types/${slug}${params}`);
+  };
+
```

Comment:
Do we need this now if you are using useQuery in the modal? I think the publish action will call fetchData in its onSuccess handler to fetch the data for the ListView table. (But I have some doubts about the useQuery in the modal)

Reply:
I agree that we need to refresh the data, but this feel a little bit manual to me.

Do you think it could be realistic to change the [list view fetching logic](https://github.com/strapi/strapi/blob/4522fcab25fb1dce1591acc477b52322cdd9b14f/packages/core/admin/admin/src/content-manager/pages/ListView/index.js#L324-L341) to use react-query instead of an effect? If we did, we'd then be able to use [query invalidation](https://tanstack.com/query/v4/docs/react/guides/query-invalidation) to refresh the list view instead of making our own callback